dll/win32/browseui/lang/it-IT.rc

```diff
@@ -150,7 +150,7 @@ BEGIN
     DEFPUSHBUTTON "Sea&rch", IDC_SEARCH_BUTTON, 80, 110, 40, 15, WS_GROUP | WS_TABSTOP | WS_VISIBLE
     PUSHBUTTON "&Stop", IDC_SEARCH_STOP_BUTTON, 80, 110, 40, 15, WS_GROUP | WS_TABSTOP
 
-    AUTOCHECKBOX "Search Hidden Files and Folders",
+    AUTOCHECKBOX "Cercca file e cartelle nascoste:",
```

Comment:
Unlikely that MS ends an autocheckbox with `:` Please prove that with a screenshot of italian MS Windows, or strip it.

Reply:
Can you please request a change here? (gonna strip it as indeed it hasn't the ':' in the italian veesion and 'Cerca' has a typo, being 'Cercca'